### PR TITLE
(docs/dualstack): v1.17 updates

### PR DESCRIPTION
**Pull request by lachie83**
1.17 docs updates for dualstack based on [issues/563](https://github.com/kubernetes/enhancements/issues/563)

Signed-off-by: Lachlan Evenson <lachlan.evenson@microsoft.com>

/milestone 1.17
/hold
/sig network

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
>
> For pull requests on Chinese localization, set Base Branch to release-1.16
> Feel free to ask questions in #kubernetes-docs-zh
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by irvifa** [comment]
Hi @lachie83 
I'm a docs release team shadow. This is a kind reminder since we're approaching the deadline for PR to be ready to be reviewed by Nov 15, 2019. Kindly update this PR, thanks a lot..

**Comment by lachie83** [comment]
@irvifa - This PR is ready for review.

**Comment by irvifa** [comment]
Thank you so much mate!
/cc @daminisatya

File: content/en/docs/tasks/network/validate-dual-stack.md [hunk 1]
```diff
@@ -58,6 +58,41 @@ kubectl get pods pod01 -o go-template --template='{{range .status.podIPs}}{{prin
 a00:100::4
 ```
 
+You can also validate Pod IPs using the Downward API via the `status.podIP` fieldPath. The following snippet demonstrates how you can expose the Pod IPs via an environment variable called `MY_POD_IPS` within a container.
```
Comment (aramase) [inline content/en/docs/tasks/network/validate-dual-stack.md]: nit: `status.podIPs`

**Comment by daminisatya** [comment]
@lachie83 Just a reminder about the last Docs deadline - 22nd Nov, by which this PR needs to be merged!

**Comment by daminisatya** [comment]
@lachie83 Can you please suggest a reviewer who can give a technical review done?

File: content/en/docs/tasks/network/validate-dual-stack.md [hunk 2]
```diff
@@ -58,6 +58,41 @@ kubectl get pods pod01 -o go-template --template='{{range .status.podIPs}}{{prin
...
+        env:
+        - name: MY_POD_IPS
+          valueFrom:
+            fieldRef:
+              fieldPath: status.podIPs
+```
+
+The following command prints the value of the `MY_POD_IPS` environment variable from within a container. The value is a comma separated list that corresponds to the Pod's IPv4 and IPv6 addresses.
+```shell
+kubectl exec -it pod01 -- env | grep MY_POD_IPS
```
Comment (khenidak) [inline content/en/docs/tasks/network/validate-dual-stack.md]: not:use `set` not `env`. AFIAK `set` is a builtin, `env` is not.

Comment (khenidak) [inline content/en/docs/tasks/network/validate-dual-stack.md]:
```suggestion
kubectl exec -it pod01 -- set | grep MY_POD_IPS
```

**Comment by daminisatya** [comment]
thank you @khenidak for the review. 

/lgtm
/approve

**Pull request merged**